### PR TITLE
disable multi device link and show warnings placeholder

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: ts/components/session/ActionsPanel.tsx [hunk 1]
```diff
@@ -56,6 +57,29 @@ export class ActionsPanel extends React.Component<Props, State> {
           },
           'refreshAvatarCallback'
         );
+        setTimeout(async () => {
+          const hasMultipleDevices =
```
Comment (Mikunj) [inline ts/components/session/ActionsPanel.tsx:61]:
```
const currentDevice = ...;
const secondaryDevices = await MultiDeviceProtocol.getSecondaryDevices(currentDevice);
const isSecondary = secondaryDevices.contains(currentDevice) || window.textsecure.storage.get('isSecondaryDevice')
```

Comment (Bilb) [reply ts/components/session/ActionsPanel.tsx:69]: fixed

File: ts/components/session/ActionsPanel.tsx [hunk 2]
```diff
@@ -59,18 +59,33 @@ export class ActionsPanel extends React.Component<Props, State> {
           'refreshAvatarCallback'
         );
         setTimeout(async () => {
+          const disabledMultiDeviceCountDb = await window.Signal.Data.getItemById(
```
Comment (Mikunj) [inline ts/components/session/ActionsPanel.tsx:62]: We can call `getItemById` directly from tx

File: ts/components/session/ActionsPanel.tsx [hunk 3]
```diff
@@ -59,18 +59,33 @@ export class ActionsPanel extends React.Component<Props, State> {
...
+            'disabledMultiDeviceCount'
+          );
+          const disabledMultiDeviceCount =
+            Number(disabledMultiDeviceCountDb?.value) || 0;
+          const data = {
+            id: 'disabledMultiDeviceCount',
+            value: String(disabledMultiDeviceCount + 1),
+          };
+          await window.Signal.Data.createOrUpdateItem(data);
```
Comment (Mikunj) [inline ts/components/session/ActionsPanel.tsx:71]: Same here

Comment (Bilb) [reply ts/components/session/ActionsPanel.tsx]: fixed

File: ts/components/session/ActionsPanel.tsx (hunk 2 above)
Comment (Bilb) [reply ts/components/session/ActionsPanel.tsx]: fixed

**Pull request merged**